keyboards/clawsome/bookerboard/rules.txt

```diff
@@ -0,0 +1,37 @@
+# MCU name
```

Comment:
This file should be deleted -- `rules.mk` will handle all this.

Reply:
I deleted it in my GitHub desktop, then committed it